lib/cloud/settings.js

```diff
@@ -78,5 +78,29 @@ export const settings = {
   forceAsset: process.env.FORCE_ASSET || false,
   databaseURL: process.env.DATABASE_URL ||
     'postgres://postgres@db/postgres?sslmode=disable',
+
+  assetStore: process.env.ASSET_STORE || 'fs',
+
+  // FS asset settings
+  assetStoreSecret: process.env.ASSET_STORE_SECRET || 'asset-secret',
+  assetStoreURLPrefix: process.env.ASSET_STORE_URL_PREFIX ||
+    'http://127.0.0.1:3000/files',
+  assetStoreURLExpireDuration:
+    parseInt(process.env.ASSET_STORE_URL_EXPIRE_DURATION) || 3600,
+
+  // S3 asset settings
+  assetStoreS3AccessKey: process.env.ASSET_STORE_ACCESS_KEY || '',
+  assetStoreS3SecretKey: process.env.ASSET_STORE_SECRET_KEY || 'asset-secret',
+  assetStoreS3Bucket: process.env.ASSET_STORE_BUCKET || 'bucket-name',
+  assetStoreS3Region: process.env.ASSET_STORE_REGION || 'eu-west-1',
```

Comment:
Although we should not relay on default. I think the common default is `us-east-1`?